installer/app/src/views/azure-create-app-tutorial.js.jsx

```diff
@@ -4,164 +4,261 @@ import Colors from './css/colors';
 import Sheet from './css/sheet';
 import InputSelection from './input-selection';
 
+var tutorialSlides = [''];
+
+tutorialSlides.push(
+    <div id="tutorial-01">
+        <img src={AssetPaths["azure-1.gif"]} style={{
+                width: 699,
+                height: 378
+            }} /> 
+        <p><a href="https://manage.windowsazure.com">Sign into the Azure Management Portal</a> and select the "Active Directory" navigation item on the left.</p>
+    </div>
+);
+
+tutorialSlides.push(
+    <div id="tutorial-02">
+        <img src={AssetPaths["azure-2.gif"]} style={{
+                width: 699,
+                height: 378
+            }} />
+        <p>Click on "Default Directory" (or the one you want to use) and select the "Applications" navigation tab.</p>
+    </div>
+);
+
+tutorialSlides.push(
+    <div id="tutorial-03">
+    <img src={AssetPaths["azure-3.gif"]} style={{
+                width: 699,
+                height: 378
+            }} />
+        <p>Click the "Add" button at the bottom and select "Add an application my organization is developing". For the name, choose "flynn-installer" (or something else), then select "Native Client Application".</p>
+    </div>
+);
+
+tutorialSlides.push(
+    <div id="tutorial-04">
+    <img src={AssetPaths["azure-4.gif"]} style={{
+                width: 699,
+                height: 378
+            }} />
+        <p>As Redirect URI, use the value below, and create the application by hitting the checkmark in the lower right.</p>
+    </div>
+);
```

Comment:
`var RedirectURISlideIndex = tutorialSlides.length-1` and ditto for all other indices referenced elsewhere